src/features/canvass/l10n/messageIds.ts

```diff
@@ -9,6 +9,56 @@ export default makeMessages('feat.canvass', {
     location: m('Untitled location'),
   },
   households: {
+    bulkEditHouseholds: {
+      floorLabel: m('Edit floor'),
+      header: m<{ numHouseholds: number }>(
+        'Edit {numHouseholds, plural, one {1 household} other {# households}}'
+      ),
+      saveButtonLabel: m<{ numHouseholds: number }>(
+        'Save {numHouseholds, plural, one {1 household} other {# households}}'
+      ),
+    },
+    colorPicker: {
+      colorNames: {
```

Comment:
I really like that we’re showing human-readable names when selecting a color!  I'm sure it will make organizing the canvassers so much easier :)